### DISCUSSION TITLE
Allowing apps on MAC OS

### DISCUSSION THREAD

**Discussion by fkm97** [SafeExamBrowser/seb-mac#351] [Q&A]
Hi,

I was just wondering how would I set up my SEB config file to allow Read&Write to be used in MAC? I got it working on Windows and tried several different options for MAC but they are not working, some assistance on this would be greatly appreciated. I am using the latest version of SEB.

![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/72949739/4650b4ce-466f-40d0-becf-6e11d539033f)

**Comment by danschlet** [comment]
See https://github.com/SafeExamBrowser/seb-mac/discussions/280. Currently not supported, see also manual.
